DataFormats/OnlineMetaData/src/classes_def.xml

```diff
@@ -0,0 +1,19 @@
+<lcgdict>
+  <class name="DCSRecord" ClassVersion="0">
+    <version ClassVersion="0" checksum="2059251589"/>
```

Comment:
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning
> Version numbers 0, 1, and 2 have special meaning to ROOT and should not be used as version numbers. By convention, CMS will use 3 as the initial version number of a class which has never been stored before.

Reply:
IIUC, partitionNames_ should be declared as transient

Reply:
How do I declare partitionNames_ transient?